cluster/cloudimages/k8s-ebs-jessie-amd64-hvm.yml

```diff
@@ -0,0 +1,66 @@
+---
+name: k8s-1.2-debian-{system.release}-{system.architecture}-{provider.virtualization}-{%Y}-{%m}-{%d}-ebs
+provider:
+  name: ec2
+  virtualization: hvm
+  enhanced_networking: simple
+  description: Kubernetes 1.2 Base Image - Debian {system.release} {system.architecture}
+bootstrapper:
+  workspace: /target
+  # tarball speeds up development, but for prod builds we want to be 100% sure...
+  # tarball: true
+system:
+  release: jessie
+  architecture: amd64
+  # We use grub, not extlinux.
+  # See https://github.com/andsens/bootstrap-vz/issues/182
+  # extlinux makes it harder to modify boot args, and may have reboot problems
+  # bootloader: extlinux
+  bootloader: grub
+  charmap: UTF-8
+  locale: en_US
+  timezone: UTC
+volume:
+  backing: ebs
+  partitions:
+    type: msdos
+    root:
+      filesystem: ext4
+      size: 8GiB
+packages:
+  mirror: http://cloudfront.debian.net/debian
+  install:
+    # cloud-initramfs-growroot will resize the master partition on boot
+    - cloud-initramfs-growroot
+    # These packages would otherwise be installed during first boot
+    - aufs-tools
+    - curl
+    - python-yaml
+    - git
+    - nfs-common
+    - bridge-utils
+    - logrotate
+    - socat
+    - python-apt
+    - apt-transport-https
+    - unattended-upgrades
+    - lvm2
+    - btrfs-tools
+plugins:
+  cloud_init:
+    metadata_sources: Ec2
+    username: admin
+  ntp: {}
```

Comment:
I think it would be good to add a default set of ntp servers since this image will likely be used by someone in production